locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed using
```

Comment:
@fhemberger @ljharb I've added some information in general (very short) about nvm and install instructions. Let me know what you think. Thanks